src/foremast/app/create_app.py

```diff
@@ -87,11 +86,26 @@ def create_app(self, pipeline_configs=None):
         """
         self.appinfo['accounts'] = self.get_accounts()
         self.log.debug('App info:\n%s', pformat(self.appinfo))
-
+        self.add_to_instance_links(pipeline_configs=pipeline_configs)
         jsondata = get_template(template_file='infrastructure/app_data.json.j2',
                                 appinfo=self.appinfo, pipeline_configs=pipeline_configs)
 
         wait_for_task(jsondata)
 
         self.log.info("Successfully created %s application", self.appname)
         return
+
+    def add_to_instance_links(self, pipeline_configs):
+        """Appends on existing instance links
+
+        Args:
+            pipeline_configs (dict): A dictionary containing the pipline.json info
+
+        Returns:
+            pipeline_configs (dict): The dictionary now containing instance links
+        """
+        if pipeline_configs['type'] == 'ec2':
+            for key, value in LINKS.items():
+                if key not in pipeline_configs['instance_links']:
+                    pipeline_configs['instance_links'][key] = value
+        return pipeline_configs
```

Comment:
I am iffy on this logic. I dont like that we are mutating a dictionary like this out of band. You do not actually use the return here. 

I think it might make more sense to make a copy of the dictionary and return that. Or to make the pipeline_configs a class attribute and modify `self.configs` without passing it around to functions